wiki/Tournaments/otst!/2020/en.md

```diff
@@ -123,6 +150,62 @@ The osu! TV Size Tournament 2020 is run by various community members.
   - [IOSYS - Cirno no Perfect Sansuu Kyoushitsu (alacat) \[Insane\]](https://osu.ppy.sh/beatmapsets/392215#osu/952069)
   - [Primastella - Koigokoro (Luerxa) \[Delis' Insane\]](https://osu.ppy.sh/beatmapsets/127712#osu/323875)
 
+## Match results
+
+### Qualifiers
+
+| Seed | Team | Rate |
```

Comment:
can round the rate to two digits after decimal?

Reply:
Sure do, was asking about this before commited but no response.